tests/test_ManifestReader/test_manifestfilereader_get_manifest.py

```diff
@@ -82,14 +90,16 @@
 def test_get_manifest(
     json_file, runs_length, app_length, orc_length, ens_length, return_type
 ):
-    try:
-        manifest_file_reader = ManifestFileReader(json_file)
+    if return_type == MalformedManifestError:
```

Comment:
Could we just copy-paste this thing and make a new test for only your `MalformedManifestError` conditions? It avoids the if/else in a single test and makes the test more readily point toward what the regression is.

Reply:
definitely!